packages/contract-method-name/cfx-name.js

```diff
@@ -0,0 +1,34 @@
+import {Conflux} from 'js-conflux-sdk'
+import {CFX_SCAN_TESTNET_DOMAIN, CFX_SCAN_MAINNET_DOMAIN} from './constance'
+import fetchHelper from './util/fetch-helper'
+
+// return conflux scan domain. Param networkType can be mainnet or testnet
+export const getCFXScanDomain = networkType => {
+  return networkType === 'mainnet'
+    ? CFX_SCAN_MAINNET_DOMAIN
+    : CFX_SCAN_TESTNET_DOMAIN
+}
+export const getCFXAbi = async (address, networkType) => {
+  const scanDomain = getCFXScanDomain(networkType)
+  return await fetchHelper(
+    `${scanDomain}/v1/contract/${address}?fields=abi`,
+    'GET',
+  )
+}
+
+export const getCFXContractName = async (
+  address,
+  transactionData,
+  networkType,
+) => {
+  try {
+    const response = await getCFXAbi(address, networkType)
+    const abi = JSON.parse(response.abi)
+    return new Conflux()
+      .Contract({abi, address})
+      .abi.decodeData(transactionData)
```

Comment:
@Pana need help reviewing the usage of js sdk
will this throw error if the `transactionData` has address and there's no input netid?

Reply:
SDK will not throw error for this situation, it will return a hex40 format address